obstacle_stop_planner_refine/test/obstacle_stop_planner_node_launch_test.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2021 the Autoware Foundation
```

Comment:
Is this license correct? If you referred to some code in AWF, please write the link?

Reply:
I referred Autoware.Auto and add some modification.
https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto/-/blob/master/src/planning/lane_planner_nodes/test/lane_planner_node_launch.test.py